src/machines/modelingMachine.ts

```diff
@@ -1,6 +1,6 @@
-import { PathToNode } from 'lang/wasm'
+import { PathToNode, VariableDeclarator } from 'lang/wasm'
 import { engineCommandManager } from 'lang/std/engineConnection'
-import { isReducedMotion } from 'lang/util'
+import { isReducedMotion } from 'lib/utils'
 import {
   Axis,
```

Comment:
This is the state diagram, about as big as I could make it
![image](https://github.com/KittyCAD/modeling-app/assets/29681384/e4778b20-9f57-4155-a7e8-cd2d7d02fc19)